include/kernel.h

```diff
@@ -19,6 +19,7 @@
 #include <limits.h>
 #include <stdbool.h>
 #include <toolchain.h>
+#include "debug/object_tracing_common.h"
```

Comment:
It is currently required for the LIFO/FIFO macros in kernel.h, given a migration of the macros from "object_tracing_common.h" it would look better if the new/existing trace header was included if tracing is enabled.

Reply:
There is now a tracing_macros.h file in the tracing directory which is included in this spot instead.